.github/workflows/build_android.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     - name: Setup Android SDK
       run: |
-        $ANDROID_SDK_ROOT/tools/bin/sdkmanager --install "ndk;21.4.7075529"
+        $ANDROID_SDK_ROOT/tools/bin/sdkmanager --install "ndk;23.1.7779620"
         $ANDROID_SDK_ROOT/tools/bin/sdkmanager --install "cmake;3.18.1"
```

Comment:
This is an old version of CMake so be careful, some things we do require 3.19+

Reply:
OK, will check the version and update.